source/fundamentals/connection.txt

```diff
@@ -152,29 +152,31 @@ URI to specify the behavior of the client.
      - integer
      - ``0``
      - Specifies the maximum amount of time a connection can remain idle
-       in the connection pool before being removed and closed.
+       in the connection pool before being removed and closed. The ``0``
+       default value indicates that there is no idle time allowance.
```

Comment:
I suggest using the description from the Go driver [options package documentation](https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo/options#ClientOptions.SetMaxConnIdleTime):

> The default is 0, meaning a connection can remain unused indefinitely.

The default means that the driver never considers connections that have not been used to time out.